Controller/Adminhtml/Email/Test.php

```diff
@@ -23,21 +24,27 @@ class Test extends \Magento\Backend\App\Action
      */
     protected $_helper;
 
+    /** @var StoreManagerInterface */
+    public $storeManager;
```

Comment:
Shouldn't it be private or at least protected?

Reply:
must be protected